website/siteConfig.js

```diff
@@ -4,6 +4,15 @@
 const repoUrl = 'https://github.com/scalameta/scalafmt';
 const baseUrl = '/scalafmt/';
 
+const users = [
+  {
+    caption: 'buildo',
+    image: 'https://www.buildo.io/static/media/logo.60e4e7fc.svg',
+    infoLink: 'https://www.buildo.io',
+    pinned: true,
+  },
+];
```

Comment:
I've picked a (not-so-random) company just to showcase this feature.

I think it makes sense for Scalafmt to showcase its users! We can start with a shortlist, then tweet about it and have people to add themselves (the edit button is already in place in the full "users" page.

Reply:
We can start with this list here https://github.com/scalameta/scalafmt/blob/72def4a91ae4b68d6d1bcfc17669b7f1ae663b47/readme/src/main/scala/org/scalafmt/readme/Adopters.scala

We can then ping the people who contributed the original PR to that file to update the logo URL. Let's not add the logos ourselves, since some companies have strict policies about where their logo is used.